app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -84,6 +84,11 @@
     private Place place;
 
     private boolean isExpanded = true;
+    private boolean showNearbyFound;
```

Comment:
Since a few years our rule is to include Javadoc for any new added field.
I know the other fields do not have javadoc, but please add javadoc for any new field.
Thanks for your understanding! :-)